app/src/main/cpp/skyline/jvm.cpp

```diff
@@ -59,7 +59,12 @@ namespace skyline {
           initializeControllersId(environ->GetMethodID(instanceClass, "initializeControllers", "()V")),
           vibrateDeviceId(environ->GetMethodID(instanceClass, "vibrateDevice", "(I[J[I)V")),
           clearVibrationDeviceId(environ->GetMethodID(instanceClass, "clearVibrationDevice", "(I)V")),
-          getVersionCodeId(environ->GetMethodID(instanceClass, "getVersionCode", "()I")) {
+          showKeyboardId(environ->GetMethodID(instanceClass, "showKeyboard", "(Ljava/nio/ByteBuffer;Ljava/lang/String;)Lemu/skyline/applet/swkbd/SoftwareKeyboardDialog;")),
+          waitForSubmitOrCancelId(environ->GetMethodID(instanceClass, "waitForSubmitOrCancel", "(Lemu/skyline/applet/swkbd/SoftwareKeyboardDialog;)[Ljava/lang/Object;")),
+          closeKeyboardId(environ->GetMethodID(instanceClass, "closeKeyboard", "(Lemu/skyline/applet/swkbd/SoftwareKeyboardDialog;)V")),
+          showValidationResultId(environ->GetMethodID(instanceClass, "showValidationResult", "(Lemu/skyline/applet/swkbd/SoftwareKeyboardDialog;ILjava/lang/String;)I")),
+          getVersionCodeId(environ->GetMethodID(instanceClass, "getVersionCode", "()I")),
+          getIntegerValueId(environ->GetMethodID(environ->FindClass("java/lang/Integer"), "intValue", "()I")) {
```

Comment:
Update the whole designated initializer (including members not introduced by this PR) to use `{...}` rather than `(...)`.